release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1446,6 +1441,11 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
 ==== External provisioner for AWS EFS (Technology Preview) feature has been removed
 The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview feature has been removed and is no longer supported.
 
+[id="ocp-4-6-tls-common-name"]
+==== TLS verification falling back to the Common Name field
+
+The behavior of falling back to the Common Name field on X.509 certificates as a host name when no Subject Alternative Names are present is removed. Certificates must properly set the Subject Alternative Names field.
```

Comment:
```suggestion
The behavior of falling back to the *Common Name* field on X.509 certificates as a host name when no Subject Alternative Names are present is removed. Certificates must properly set the *Subject Alternative Names* field.
```
Per the IBM Style Guide, we should be bolding field names. 

Subject Alternative Names are capitalized like acronyms. If it is not a proper noun then it needs to be in lower case. If it is a field, then it needs to be bolded and identified as a field.

Reply:
Addressed.